src/foam/nanos/crunch/lite/Capable.js

```diff
@@ -93,6 +95,24 @@ foam.INTERFACE({
             }
           `
         }));
+        cls.methods.push(foam.java.Method.create({
+          name: 'getCapablePayloadDAO',
+          type: 'foam.dao.DAO',
+          visibility: 'default',
+          args: [
+            { name: 'x', type: 'X' }
+          ],
+          body: `
+            DAO capableDAO = new CapableAdapterDAO.Builder(x)
+              .setCapable(this)
+              .setOf(CapablePayload.getOwnClassInfo())
+              .build();
+            x = x.put("capableDAO", capableDAO);
```

Comment:
Do we need this still? It worked on mine without it unless it should be there.

Reply:
I left it there by accident, but this is more future-proof since RulerDAO takes a daoKey so maybe we should leave it